README.md

```diff
@@ -1,2 +1,27 @@
-# PiWeb Import Plugins
-Space to try documentation of Auto Importer plugins
+ZEISS PiWeb-Import-Sdk
+=========
+
+[![Build on develop](https://github.com/ZEISS-PiWeb/PiWeb-Import-Sdk/actions/workflows/develop.yml/badge.svg?branch=develop&event=push)](https://github.com/ZEISS-PiWeb/PiWeb-Import-Sdk/actions/workflows/develop.yml)
+[![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
+[![Nuget](https://img.shields.io/nuget/v/Zeiss.PiWeb.Import.Sdk?style=flat&logo=nuget)](https://www.nuget.org/packages/Zeiss.PiWeb.Import.Sdk/)
+
+The PiWeb-Import-Sdk is the basic framework for creating your own import plugins for the PiWeb Auto Importer. By creating plugins, it is possible to automate the data import of your own custom file formats or even import data from other sources like Rest-APIs or databases.
+
+### What is the PiWeb Auto Importer?
```

Comment:
Why do you start on sub-headline "3", is it because of styling?

Reply:
Yes I did not want the headline to be too large.